v1.16/Dockerfile

```diff
@@ -1,9 +1,5 @@
 FROM alpine:3.18
 
-ENV FLUENTD_VERSION 1.16.2
```

Comment:
Is it really safe to delete the `env` variable?

Reply:
I'm not sure either, so please keep it in case something builds on it.

Reply:
i see it as a possible source for inconsistencies, because this variable won´t get updated by renovate.